graylog2-web-interface/src/pages/LoginPage.jsx

```diff
@@ -14,80 +11,73 @@ import AuthThemeStyles from 'theme/styles/authStyles';
 import CombinedProvider from 'injection/CombinedProvider';
 import LoadingPage from './LoadingPage';
 
-const { SessionStore, SessionActions } = CombinedProvider.get('Session');
+const { SessionActions } = CombinedProvider.get('Session');
 
-const LoginPage = createReactClass({
-  displayName: 'LoginPage',
-  mixins: [Reflux.connect(SessionStore), Reflux.ListenerMethods],
+const LoginPage = () => {
+  const [didValidateSession, setDidValidateSession] = useState(false);
+  const [lastError, setLastError] = useState(false);
```

Comment:
I think using `false` as default `lastError` is not the best thing, we should probably consider using `undefined` or `''`.

Reply:
I had this fixed at one point, guess i overwrote myself on a rebase